packages/app/obojobo-document-json-parser/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "dependencies": {
+    "obojobo-document-engine": "^4.0.1"
+  },
+  "devDependencies": {
+    "lint-staged": "^6.1.1"
+  },
+  "name": "json-to-xml-parser",
+  "version": "4.0.1",
+  "license": "AGPL-3.0-only",
+  "main": "",
+  "scripts": {
+    "test": "TZ='America/New_York' jest --verbose",
+    "test:ci": "TZ='America/New_York' CI=true jest --ci --coverage --coverageReporters text-summary cobertura",
+    "convertJsonToXml": "node ./bin/json2xml.js examples/test-object.json > test-object.xml",
```

Comment:
Suggest adding a `bin` directive to make the xml converter scripts easier to use. See obojobo-express/package.json